inst/extdata/sectoral/mappingCEDS2025toREMIND.csv

```diff
@@ -0,0 +1,62 @@
+1B2_Fugitive-petr;Energy|Supply|Fuel Production;Energy Sector;In the future REMIND emissions accounting should separate fugitive from combustion emissions to be in line with IPCC accounting, once this is done in REMIND reporting -> adapt here;
+1B2b_Fugitive-NG-distr;Energy|Supply|Fuel Production;Energy Sector;In the future REMIND emissions accounting should separate fugitive from combustion emissions to be in line with IPCC accounting, once this is done in REMIND reporting -> adapt here;
+1B2b_Fugitive-NG-prod;Energy|Supply|Fuel Production;Energy Sector;In the future REMIND emissions accounting should separate fugitive from combustion emissions to be in line with IPCC accounting, once this is done in REMIND reporting -> adapt here;
+1B2d_Fugitive-other-energy;Energy|Supply|Fuel Production;Energy Sector;In the future REMIND emissions accounting should separate fugitive from combustion emissions to be in line with IPCC accounting, once this is done in REMIND reporting -> adapt here;
+2A1_Cement-production;Industrial Processes|Non-Metallic Minerals;Industrial Sector;;
+2A2_Lime-production;Industrial Processes|Non-Metallic Minerals;Industrial Sector;;
+2Ax_Other-minerals;Industrial Processes|Other;Industrial Sector;;
+2B_Chemical-industry;Industrial Processes|Chemicals;Industrial Sector;;
+2B2_Chemicals-Nitric-acid;Industrial Processes|Chemicals;Industrial Sector;;
+2B3_Chemicals-Adipic-acid;Industrial Processes|Chemicals;Industrial Sector;;
+2C1_Iron-steel-alloy-prod;Industrial Processes|Steel;Industrial Sector;;
+2C3_Aluminum-production;Industrial Processes|Other;Industrial Sector;;
+2C4_Non-Ferrous-other-metals;Industrial Processes|Other;Industrial Sector;;
+2D_Chemical-products-manufacture-processing;Industrial Processes|Chemicals;Solvents Production and Application;;
+2D_Degreasing-Cleaning;Industrial Processes|Other;Solvents Production and Application;;
+2D_Other-product-use;Industrial Processes|Other;Solvents Production and Application;;
+2D_Paint-application;Industrial Processes|Other;Solvents Production and Application;;
+2H_Pulp-and-paper-food-beverage-wood;Industrial Processes|Other;Industrial Sector;;
+2L_Other-process-emissions;Industrial Processes|Other;Industrial Sector;;
+3B_Manure-management;Agriculture;Agriculture;;
+3D_Rice-Cultivation;Agriculture;Agriculture;;
+3D_Soil-emissions;Agriculture;Agriculture;;
+3E_Enteric-fermentation;Agriculture;Agriculture;;
+3I_Agriculture-other;Agriculture;Agriculture;;
+5A_Solid-waste-disposal;Waste;Waste;;
+5C_Waste-combustion;Waste;Waste;;
+5D_Wastewater-handling;Waste;Waste;;
+5E_Other-waste-handling;Waste;Waste;;
+6A_Other-in-total;Industrial Processes|Other;Other;"Not sure what this is, put it to ""industrial processes"" category for now";Will be explicitly set to zero so do not map to any sector (see here: https://github.com/iiasa/emissions_harmonization_historical/issues/32)
```

Comment:
```suggestion
6A_Other-in-total;Industrial Processes|Other;Industrial Sector;"Not sure what this is, put it to ""industrial processes"" category for now";
```
Just realized that I mixed up `6A_Other-in-total` and `6B_Other-not-in-total` (the latter does not even appear here, which makes sense, as it is apparently [not considered in CEDS](https://github.com/iiasa/emissions_harmonization_historical/issues/32)). For `6A_Other-in-total` the IAMC sector should be `Industry` according to [this spreadsheet](https://docs.google.com/spreadsheets/d/1HKfl21NX8wohPEHpHlmf5XithknPw8zjLpVe4MiOU9U/edit?gid=1730258315#gid=1730258315)